tests/examples/examples_report_plugin.py

```diff
@@ -1,13 +1,12 @@
 from __future__ import absolute_import, print_function
 
+import io
 import os
+from os.path import abspath, dirname, exists, expanduser, expandvars, join, pardir
```

Comment:
almost everywhere else we import individual funcs from `os.path` in fixing up this conflict I made that change as well.